src/main/java/com/playonlinux/ui/impl/javafx/installwindow/InstallWindow.java

```diff
@@ -242,10 +243,10 @@ private void setUpEvents() throws PlayOnLinuxError {
                 playOnLinuxError.printStackTrace();
             }
         });
```

Comment:
Using the KeyReleased event instead of the KeyPressed event fixes that the Title-Filter was hanging behind by one char.